whois-commons/src/main/resources/applicationContext-commons.xml

```diff
@@ -13,69 +13,6 @@
                            http://www.springframework.org/schema/cache
                            http://www.springframework.org/schema/cache/spring-cache.xsd">
 
-    <!-- TODO: this is loaded many many times on context init, which slows down startup process -->
-    <!-- @Transactional CAN be used on private methods with mode="aspectj". However, annotations on interfaces will NOT work! -->
-    <tx:annotation-driven mode="aspectj"/>
-
-    <context:spring-configured/>
     <context:component-scan base-package="net.ripe.db.whois.common"/>
 
-    <context:annotation-config/>
```

Comment:
Do we need `AnnotationConfigApplicationContext` to replace `<context:annotation-config>` ?
i.e. to support non-Spring (Jakarta) annotations

Reply:
applicationContext-whois-base.xml has this already <context:annotation-config>